modules/nf-scil/reconst/diffusivitypriors/main.nf

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
Very nice work ! All clean. Although, I maybe not getting the full inner workings of the script. You either compute :
- per subject priors using Noddi
- create an average prior from a precomputed set ?

If so, I propose to slightly change the behavior by also computing the average priors on the Noddi output.

Reply:
Tell me if the changes to the script are what you wanted.